internal/display/apis.go

```diff
@@ -110,3 +155,35 @@ func makeScopeView(scope *management.ResourceServerScope) *scopeView {
 		Description: auth0.StringValue(scope.Description),
 	}
 }
+
+func truncateScopes(scopes []*management.ResourceServerScope) *string {
```

Comment:
<img width="712" alt="Screen Shot 2021-03-22 at 13 51 50" src="https://user-images.githubusercontent.com/5055789/112027325-d63f9f80-8b15-11eb-9ced-1308e6d3e40e.png">

Scopes that fit shouldn't be truncated.
Also let's not display the hint if the scopes are not truncated.

Reply:
🤦 good catch. Refactoring this now to bring the hint into the display logic instead of the cli logic.